src/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java

```diff
@@ -106,7 +107,6 @@
 import org.zaproxy.zap.view.HttpPanelManager.HttpPanelViewFactory;
 import org.zaproxy.zap.view.SiteMapListener;
 import org.zaproxy.zap.view.SiteMapTreeCellRenderer;
- 
```

Comment:
In general whitespace or formatting changes should be in their own PR. They seem pretty minimal in this PR, but when mixed in they make actually code/functional changes harder to spot. 
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting